### ISSUE TITLE
Application page : Initial screen 

### ISSUE THREAD

**Issue by elhaw** [m3ntorship/m3ntorship.com-v2#54]
**Story**
As a user when I navigate to the application form I should see two tabs ( mentor and mentee) with empty content.
[design link](https://www.figma.com/file/5KpDnPPmrZhudGESduPhq3/M3ntorship-Website?node-id=1089%3A8106)

**functionality**
- A user can switch between the mentor and mentee form by clicking on the associated tab
- The selected tab style should be changed as per the design 
- The progress bar should represent the current step 
- Click on the next button and go to the next step
- The previous button will appear after step 2 and clicking on it will navigate to the previous step 
- The title of the form will change after a user select the type 
- Click on the cancel button should show the UI of the first image attached below 👇 

**Acceptance criteria**

<img width="728" alt="Screen Shot 2022-08-21 at 10 26 01 PM" src="https://user-images.githubusercontent.com/8650435/185809550-7647c243-37fb-4335-b134-2f9b9d46c28a.png">
<img width="574" alt="Screen Shot 2022-08-21 at 10 26 23 PM" src="https://user-images.githubusercontent.com/8650435/185809587-d1624e1a-f725-4920-ad9a-b6482bc4dacf.png">
<img width="501" alt="Screen Shot 2022-08-21 at 11 09 16 PM" src="https://user-images.githubusercontent.com/8650435/185811034-5689d1ec-0235-4eb4-8610-befe7438ca80.png">

**Note**

we will add form qouestions  in a separate task
